app/javascript/packs/dashboard.scss

```diff
@@ -9,3 +9,41 @@
 @import '~@patternfly/patternfly/components/InputGroup/input-group.css';
 @import '~@patternfly/patternfly/components/OptionsMenu/options-menu.css';
 @import '~@patternfly/patternfly/layouts/Level/level.css';
+
+.dashboard-list-icon-title-layout {
```

Comment:
Can't these styles be imported from `app/javascript/src/Dashboard/styles/dashboard.scss` ?

Just to keep it clearer where they belong (i.e. what components depend on them)

Reply:
I agree, even though `webpacker` says they should be at `app/javsascript/styles`. I was looking for an intermediate solution where the new dashboard.scss makes sense.
The problem I see is our css is a bit of a mess to put it delicately so by moving everything in dashboard.scss (and other packs in the same way) I hope we can clean it up a bit. Some are also monkey-patches to accommodate to or from PF4...